### PR TITLE
Add `darkMode` option

### LINKED ISSUE #36: Add `darkMode` option
To force websites that supports it to be dark.

We can use the new Puppeteer feature for this: https://github.com/GoogleChrome/puppeteer/blob/v2.0.0/docs/api.md#pageemulatemediafeaturesfeatures

We should also force the default to be `await page.emulateMediaFeatures([{ name: 'prefers-color-scheme', value: 'light' }]);` so people get predictable results.

Should be added to https://github.com/sindresorhus/capture-website-cli too.

### PR DESCRIPTION
**Pull request by krnik**
Closes #36

Regarding `capture-website-cli`  - changes will be added after this PR is approved to minimize potential fixes.

Also, one question regarding TODO comment:
```
// TODO: Add more events from https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#event-requestfailed
```
Which events are suitable for `debug` mode?
I've added `requestfailed`. I was thinking about adding `request` but for some cases that would produce huge amount of logs.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -200,7 +201,9 @@ const captureWebsite = async (input, options) => {
 			console.log('\nPage error:', error, '\n');
 		});
 
-		// TODO: Add more events from https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#event-requestfailed
+		page.on('requestfailed', request => {
+			console.log('\nRequest failed:', request.url(), '\n');
+		});
```
Comment (sindresorhus) [inline index.js]: This is unrelated to the intent of this PR and should be removed.

**Comment by krnik** [comment]
Irrelevant changes were removed, PR in cli repo was created
https://github.com/sindresorhus/capture-website-cli/pull/19

**Comment by sindresorhus** [comment]
Thanks :)
